Clinical-Genomics/scout
# Send params to IGV view via POST request

fix #1653 , at least for the data already passed to the igv view in current master.

**How to test**:
1. Install the branch on scout stage (clinical-db)
1. Choose a recently updated case (for instance: https://scout-stage.scilifelab.se/cust002/F0000496)
1. Inspect a snv variant and make sure it looks as it should on igv
1. Inspect a sv variant and make sure it looks as it should on igv
1. Open the list of str variants and make sure the link to igv works and they are viewed correctly in this browser.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] tests executed by CR (the branch should still be on clinical-db stage if nobody changes it)
